src/js/actions/libraries.js

```diff
@@ -145,7 +145,7 @@ define(function (require, exports) {
             currentLayer = currentLayers.first();
 
         if (!currentLibrary || currentLayers.size !== 1 ||
-            !currentLayer || currentLayer.kind !== currentLayer.layerKinds.TEXT) {
+            !currentLayer || currentLayer.isTextLayer()) {
```

Comment:
Ick, while this may seem like a good idea, I don't think we should add yet another property checker function to the model. @iwehrman thoughts?